app/controllers/applications_controller.rb

```diff
@@ -17,6 +17,11 @@ def new
       alreadyApplied << a.property
     end
     @properties = (@properties - alreadyApplied) | (alreadyApplied - @properties)
+    
+    # Approach x: reordering properties based on price matching score with tenant's min_rent
+	  @properties = @tenant.recommend_properties(@properties)  # Should we call the function here or in /views/applications/new.html.erb?
+  	#####
```

Comment:
To address your comment, you should never call ruby methods in html files. Those files are for purely frontend work.